rc/.eslintrc

```diff
@@ -1,3 +1,10 @@
 {
-  "extends": ["standard", "standard-react"]
+  "extends": ["standard", "standard-react"],
+  "rules": {
+    "standard/object-curly-even-spacing": [2, "either"],
```

Comment:
I think adding the rules to `eslint-config-standard` would work similar to how [eslint-config-standard-react](https://github.com/feross/eslint-config-standard-react) works.

`eslint-standard-react` uses `eslint-plugin-react`, which is specified in `peerDependencies` on that package. So, when `eslint-standard-react` is installed, npm automatically installs the plugin too. (https://docs.npmjs.com/files/package.json#peerdependencies)

So, adding this plugin to `eslint-config-standard` should be OK as long as the plugin is added to `peerDependencies`.